server/ai_http.go

```diff
@@ -196,7 +196,7 @@ func handleAIRequest(ctx context.Context, w http.ResponseWriter, r *http.Request
 			width = int64(*v.Width)
 		}
 
-		outPixels = height * width
+		outPixels = height * width * int64(*v.NumImagesPerPrompt)
```

Comment:
@pschroedl we could add a `nil` check here aswell but I removed it since it should not occur with the code I now added at the gateway.

Reply:
I like the early check and reassignment to default if nil, particulary understanding that ultimately we will want middleware to perform the same.

Reply:
Added.